Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/ExamplesHubHomeButtonActivation.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// Shows the home button in the ToggleFeaturesPanel if the Scene System is enabled
+    /// </summary>
+    public class ExamplesHubHomeButtonActivation : MonoBehaviour
+    {
+        /// <summary>
+        /// Home button which brings the user back to examples hub
+        /// </summary>
+        [SerializeField]
+        [Tooltip("Home button which brings the user back to examples hub")]
+        private GameObject buttonHubHome = null;
+
+        private void OnEnable()
+        {
+            if (MixedRealityToolkit.IsSceneSystemEnabled == true)
```

Comment:
This seems much more like an example to me -- the scrip doesn't have tons of value beyond the example. Additionally, the ToggleFeaturesPanel prefab is used in HandInteractionExamples, isnt' it? In that case we don't want a home button. It feels like toggle features with a home button should be a separate prefab, that lives in the examples folder. Then you could move this script back to Examples folder, where I think it belongs.

Reply:
Right. Tried to think about separating the home button from the ToggleFeaturesPanel or having duplicated ToggleFeaturesPanel for the Examples Hub. The challenge was, there are many example scenes that are using ToggleFeaturesPanel. I wanted to make the example scene work for both as a standalone and as part of the Example Hub.

Reply:
Putting a separate home button for the hub is easy but make it nicely aligned with ToggleFeaturesPanel would be a challenge.

Reply:
I'll duplicate the ToggleFeaturesPanel for the example hub specific version. I'll put this info in the README file. Thanks!